### PR TITLE
Desktop portfolio 

### PR DESCRIPTION
**Pull request by mwanawabangona**
used HTML and CSS
 I created a new branch
 I fixed linters errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+                        <div class="tags">
+                          <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                          </ul>
+                        </div>
+
+                        <a href="#" class="btn-one">See project</a>
+                      </li>
```
Comment (vichuge) [inline index.html]: - [ ] It looks like some project has a different size on the list, I kindly invite you to work on it.
- [ ] The background color could improve too. 🐶 

| Original | your project |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+        cols="30"
+        rows="7"
+        class="msg"
+        required
+        placeholder="Enter your message here"
+      ></textarea>
+      <button type="submit" class="project-button send">
+        Get in Touch
+      </button>
+    </form>
```
Comment (vichuge) [inline index.html:312]: - [ ] On desktop size, the form inputs stop to use all the weight screens, look at the original template and please, fix it. 🐱

| original | your project |
| --- | --- |
| image | image |

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+}
+
+.menu {
+  display: block;
+  position: absolute;
+  right: 15px;
+  top: 15px;
+}
+
+@media only screen and (min-width: 768px) {
```
Comment (vichuge) [inline css/style.css]: - [ ] Uses media queries for the Desktop breakpoint in the CSS file. (the value for min-width property should be in range of 992px - 1100px). Please fix this. 🌵

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+  margin-bottom: 32px;
+  border: none;
+  padding: 14px;
+}
+
+.send:hover {
+  border: 1px solid white;
+}
+
+@media only screen and (max-width: 769px) {
```
Comment (vichuge) [inline css/style.css:461]: - [ ] I kindly remind you, consider the mobile phone when you work on the desktop size, this should work as usual when you finish your desktop size, following your own deployment, this is how the mobile size looks using chrome web tools. I kindly invite you to work on this. 🐦 
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.
- [ ] Please, work in your PR description following the [Professional repository rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=Write%20a%20meaningful%20title%20for%20your%20pull%20request%20and%20always%20add%20a%20short%20summary%20of%20changes%20introduced%20in%20the%20pull%20request.)
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @mwanawabangona , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**